engine/security/https.md

```diff
@@ -93,7 +93,7 @@ Now, generate the signed certificate:
     Getting CA Private Key
     Enter pass phrase for ca-key.pem:
 
-[Authorization plugins](../extend/plugins_authorization) offer more
+Authorization plugins offer more
```

Comment:
Looks like the URL has to be one level up; the page that's linked to is fetched during build of the documentation, so perhaps good to just use an absolute path;

```suggestion
[Authorization plugins](/engine/extend/plugins_authorization/) offer more
```